docs/index.md

```diff
@@ -1,7 +1,7 @@
 <p style="text-align: left;" class="build">
-    <img src="assets\images\buildwithprivacy.png">
+    <img src="assets\images\build-with-encryption.jpg">
 </p>
 
-Welcome to Obscuro. Privacy for Ethereum on a Layer 2 roll-up. No SDKs, 100% EVM.
+Welcome to Obscuro. Obscuro hyper-scales and encrypts Ethereum. Obscuro ensures the inputs, smart contract state and execution are always encrypted.
```

Comment:
personally, I prefer "No SDKs, 100% EVM." vs "Obscuro ensures the inputs, smart contract state and execution are always encrypted." in this heading.

because the latter is just repeating what the proceeding sentence states with "encrypts ethereum" but with more detail than needed for a heading.